p2p/pex/pex_reactor_test.go

```diff
@@ -450,7 +451,7 @@ func teardownReactor(book *addrBook) {
 }
 
 func createSwitchAndAddReactors(reactors ...p2p.Reactor) *p2p.Switch {
-	sw := p2p.MakeSwitch(config, 0, "127.0.0.1", "123.123.123", func(i int, sw *p2p.Switch) *p2p.Switch { return sw })
+	sw := p2p.MakeSwitch(cfg, 0, "127.0.0.1", "123.123.123", func(i int, sw *p2p.Switch) *p2p.Switch { return sw })
```

Comment:
maybe it would be better to make it more explicit by calling it `testCfg` or something ?

Reply:
That it is global state is already a smell, it's gonna vanish when all the config traces are gone from the package.